util/Migrator/DbScripts/2021-04-13_00_CipherPasswordPrompt.sql

```diff
@@ -0,0 +1,202 @@
+﻿IF COL_LENGTH('[dbo].[Cipher]', 'Reprompt') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Cipher]
+        ADD [Reprompt] TINYINT NULL;
+END
+GO
+
+UPDATE
+    [dbo].[Cipher]
+SET
+    [Reprompt] = 0
+GO
+
+ALTER TABLE
```

Comment:
Like the column check, if we're leaving this column `NOT NULL` in the final outcome, you would need to add a check on `sys.columns` on this column not already being `NOT NULL` before making this modification.